core/src/network/entry_with_header.rs

```diff
@@ -34,6 +34,7 @@ pub fn fetch_entry_with_header(
     context: &Arc<Context>,
 ) -> Result<EntryWithHeader, HolochainError> {
     let entry = fetch_entry_from_cas(address, &context)?;
+    println!("entry found {:?}", entry.clone());
```

Comment:
This needs to be a call to context.log() or deleted.

Reply:
Oh yes I thought I got them all haha. Will make those changes thank you so so much